initPose_x: 46.522
initPose_y: -5.90
initPose_th: 1.415

goalPose_x: -7.868
goalPose_y: 1.5968
goalPose_th: 3.076
